### DISCUSSION TITLE
Thank you for your contributions in April 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#12157] [Community]
We're grateful for all the fantastic contributions we received in April when we merged pull requests from

@Jaspreet-singh-1032 @EshaanAgg @kafukoM @oge1ata @practicatto @Wck-iipi @GSAprod @GarvitSinghal47 @satyamkale27 @thesujai @BabyElias @KshitijThareja and @shivansh-bhatnagar18.

One of the core Kolibri Design System components was refactored to allow for better future maintenance, Kolibri has a plenty new frontend tests, new and old bugs are fixed, Studio is a few steps closer to using Kolibri Design System, and much more. Thanks to each one of you and to everyone who has work in progress!

This month, we're finally introducing the Kolibri Contributors Community to the public. It is an emerging group of active volunteers who regularly contribute to Learning Equality's GitHub repositories. It provides new ways to connect and acknowledge your contributions. Find out more [in this post](https://github.com/learningequality/kolibri/discussions/12156) and join us!

**Comment by radinamatic** [comment]
Great job everybody, **you** make this community ❤️ ❤️ ❤️!

**Comment by rtibbles** [comment]
Thank you everybody for being part of and helping us to build this great community!
